src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/mapper/LimitedPartnershipPatchMapperTest.java

```diff
@@ -39,12 +40,17 @@ void testObjectMapperCanHandleJsonNullableFields() throws JsonProcessingExceptio
 
     @ParameterizedTest
     @CsvSource(value = {
-            // Field NOT present in the JSON - no update:
-            "{\"email\":\"test@test.com\"}$ Asset Strippers $ test@test.com",
-            // Field IS present in the JSON - set the new string value:
-            "{\"partnership_name\":\"Asset Adders\", \"email\":\"test@test.com\"}$ Asset Adders $ test@test.com"
+            // Fields NOT present in the JSON - no update:
+            "{\"email\":\"test@test.com\"}$ Asset Strippers $ test@test.com $ Scotland",
+            // Fields ARE present in the JSON - set the new string value:
+            "{\"partnership_name\":\"Asset Adders\", \"email\":\"test@test.com\", \"jurisdiction\":\"Scotland\"}$ Asset Adders $ test@test.com $ Scotland",
+            // Jurisdiction field is invalid in the JSON - set the 'Unknown' value:
+            "{\"partnership_name\":\"Asset Adders\", \"email\":\"test@test.com\", \"jurisdiction\":\"Mongolia\"}$ Asset Adders $ test@test.com $ Unknown"
     }, delimiter = '$')
-    void testMapStructMappingWhenEmailValueSentAndNameUnchanged(String incomingJson, String expectedPartnershipName, String expectedEmail)
+    void testMapStructMappingWhenEmailValueSentAndNameUnchanged(String incomingJson,
```

Comment:
May want to abbreviate this description as well. Could we say just "mapping" here?

Reply:
Done.